### PR TITLE
edb service definition

### PR DESCRIPTION
**Pull request by caggles**

### CODE REVIEW

File: docs/EDB/ServiceDefinition.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+# BC Government EDB Operator Service Definition
+
+## Service Description
+
+### Summary
+
+### Features & Functions
+
+This service is an operator built by EDB, which is used to install and update instances of Postgres in various namespaces.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:21]:
```suggestion
This Enterprise DB service is provided as one of the Shared Services on the BC Gov's DevOps Openshift 4 Platform. This service enabled product teams to use EnterpriseDB version of PostgreSQL to create HA database istances in their app. The EnterpriseDB Services is available to all teams as a free community version of EnterpriseDB or as an enterprise version for those teams that have purchased an enterprise license for EnterpriseDB. The service is enabled by an operator built by EDB and installed in OCP 4 Silver cluster, which is used to install and update instances of Postgres in various namespaces.
```

File: docs/EDB/ServiceDefinition.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+You can find further information on the operator's features and functionality here: https://docs.enterprisedb.io/cloud-native-postgresql
+
+### Eligibility & Prerequisites
+
+This service is offered to BC Government development teams building cloud native applications on the Openshift Platform.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:27]:
```suggestion
This service is offered to BC Government development teams building cloud native applications on the Openshift 4 Platform. The EnterpriseDB Services is available to all teams as a free community version of EnterpriseDB or as an enterprise version for those teams that have purchased an enterprise license for EnterpriseDB. 
```

File: docs/EDB/ServiceDefinition.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+```
+
+---
+
+# License
+
+[![Creative Commons License](https://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/)
+
+```
+Copyright 2019 Province of British Columbia
```
Comment (StevenBarre) [inline docs/EDB/ServiceDefinition.md:128]:
```suggestion
Copyright 2021 Province of British Columbia
```

File: docs/EDB/ServiceDefinition.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+# BC Government EDB Operator Service Definition
+
+## Service Description
+
+### Summary
+
+### Features & Functions
+
+This Enterprise DB service is provided as one of the Shared Services on the BC Gov's DevOps Openshift 4 Platform. This service enables product teams to use EnterpriseDB version of PostgreSQL to create HA database istances in their app. The service is enabled by an operator built by EDB and installed in OCP 4 Silver cluster, which is used to install and update instances of Postgres in various namespaces.
```
Comment (StevenBarre) [inline docs/EDB/ServiceDefinition.md:21]:
```suggestion
The Enterprise DB service is provided as one of the Shared Services on the BC Gov's DevOps OpenShift 4 Platform. This service enables product teams to use the EnterpriseDB version of PostgreSQL to create HA database instances in their app. The service is enabled by an operator built by EDB and installed in OCP 4 Silver cluster, which is used to install and update instances of Postgres in various namespaces.
```

File: docs/EDB/ServiceDefinition.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+You can find further information on the operator's features and functionality here: https://docs.enterprisedb.io/cloud-native-postgresql
+
+### Eligibility & Prerequisites
+
+This service is offered to BC Government development teams building cloud native applications on the Openshift 4 Platform. The EnterpriseDB Services is available to all teams as a free community version of EnterpriseDB or as an enterprise version for those teams that have purchased an enterprise license for EnterpriseDB.
```
Comment (StevenBarre) [inline docs/EDB/ServiceDefinition.md:27]:
```suggestion
This service is offered to BC Government development teams building cloud native applications on the OpenShift 4 Platform. The EnterpriseDB Services is available to all teams as a free community version of EnterpriseDB or as an enterprise version for those teams that have purchased an enterprise license for EnterpriseDB. 
```

File: docs/EDB/ServiceDefinition.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+[RocketChat](https://chat.developer.gov.bc.ca) is the primary mode of communication. Specifically the `#edb` channel should be used for engage the community for best practices, configuration and troubleshooting questions.
+
+For cluster wide service notifications that may impact the operator or your database, monitor the `#devops-alerts` channels in [RocketChat](https://chat.developer.gov.bc.ca/channel/devops-alerts).
+
+For teams without RocketChat access, escalation, or to talk to a person IRL contact [Olena Mitovska](mailto:olena.mitovska@gov.bc.ca), Product Owner for Platform Services, BCDevExchange, Office of the Chief Information Officer.
+
+## Service Delivery
+
+Teams can create a custom resource in order to create a EDB Postgres database in the same way they typically create other Openshift objects.
```
Comment (StevenBarre) [inline docs/EDB/ServiceDefinition.md:92]:
```suggestion
Teams can create a custom resource in order to create a EDB Postgres database in the same way they typically create other OpenShift objects.
```

File: docs/EDB/ServiceDefinition.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+Updates to your database itself will be controlled by your team.
+
+### Service Level
+
+TBD
+
+### Security Reviews
+
+EDB is covered by the existing Openshift PIA.
```
Comment (StevenBarre) [inline docs/EDB/ServiceDefinition.md:114]:
```suggestion
EDB is covered by the existing OpenShift PIA.
```

File: docs/EDB/ServiceDefinition.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+### Eligibility & Prerequisites
+
+This service is offered to BC Government development teams building cloud native applications on the Openshift 4 Platform. The EnterpriseDB Services is available to all teams as a free community version of EnterpriseDB or as an enterprise version for those teams that have purchased an enterprise license for EnterpriseDB. 
+
+### How to Request  
+
+Teams do not need to request anything from the platform team in order to make use of this operator in order to install Postgres community edition.
+If a team wishes to purchase a license in order to make use of the enterprise edition of the operator, 
+teams should contact the [Olena Mitovska](mailto:olena.mitovska@gov.bc.ca) in order to discuss options - licenses are purchased collectively for ease of administration.
```
Comment (StevenBarre) [inline docs/EDB/ServiceDefinition.md:33]:
```suggestion
they should contact [Olena Mitovska](mailto:olena.mitovska@gov.bc.ca) in order to discuss options - licenses are purchased collectively for ease of administration.
```

File: docs/EDB/ServiceDefinition.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+## Support Roles, Processes, Communications (platform ops)
+
+The team supporting this service administers the EDB operator used to install the database, but does not administer your database itself.
+
+[RocketChat](https://chat.developer.gov.bc.ca) is the primary mode of communication. Specifically the `#edb` channel should be used for engage the community for best practices, configuration and troubleshooting questions.
+
+For cluster wide service notifications that may impact the operator or your database, monitor the `#devops-alerts` channels in [RocketChat](https://chat.developer.gov.bc.ca/channel/devops-alerts).
+
+For teams without RocketChat access, escalation, or to talk to a person IRL contact [Olena Mitovska](mailto:olena.mitovska@gov.bc.ca), Product Owner for Platform Services, BCDevExchange, Office of the Chief Information Officer.
```
Comment (StevenBarre) [inline docs/EDB/ServiceDefinition.md:88]:
```suggestion
For teams without RocketChat access, escalation, or to talk to a person IRL, contact [Olena Mitovska](mailto:olena.mitovska@gov.bc.ca), Product Owner for Platform Services, BCDevExchange, Office of the Chief Information Officer.
```

**Comment by caggles** [comment]
Heads up everyone - the content of the doc has changed fairly significantly because there has been some clarification about the way the licenses work.

**Review by StevenBarre — APPROVED** [review]

File: docs/EDB/ServiceDefinition.md [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+
+## Service Description
+
+### Summary
+
+### Features & Functions
+
+The Enterprise DB service is provided as one of the Shared Services on the BC Gov's DevOps OpenShift 4 Platform. 
+This service enables product teams to use the EnterpriseDB version of PostgreSQL to create HA database instances in their app. 
+The service is enabled by an operator built by EDB and installed in OCP 4 Silver cluster, which is used to install and update instances of Postgres in various namespaces.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:23]:
```suggestion
The service is enabled by an operator built by EDB and installed in OCP 4 Silver cluster, which can be used to install and add HA capabilities to the instances of Postgres in various namespaces.
```

File: docs/EDB/ServiceDefinition.md [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+
+You can find further information on the operator's features and functionality here: https://docs.enterprisedb.io/cloud-native-postgresql
+
+Associated with the operator is the RemoteDBA (RDBA) support service, also offered by EDB.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:27]:
```suggestion
Associated with the operator is the RemoteDBA (RDBA) support service offered by the EnterpriseDB vendor under a separate license. 
```

File: docs/EDB/ServiceDefinition.md [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+### Features & Functions
+
+The Enterprise DB service is provided as one of the Shared Services on the BC Gov's DevOps OpenShift 4 Platform. 
+This service enables product teams to use the EnterpriseDB version of PostgreSQL to create HA database instances in their app. 
+The service is enabled by an operator built by EDB and installed in OCP 4 Silver cluster, which is used to install and update instances of Postgres in various namespaces.
+
+You can find further information on the operator's features and functionality here: https://docs.enterprisedb.io/cloud-native-postgresql
+
+Associated with the operator is the RemoteDBA (RDBA) support service, also offered by EDB. 
+They are involved in providing support for the operator and support for database clusters belonging to teams which have purchased their additional support packages.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:28]:
```suggestion
The Enterprise DB's RDBA team assist the Platform Services Team with providing support for the EDB operator and can also provide support for database clusters belonging to the product teams which have purchased a license for their service. 
```

File: docs/EDB/ServiceDefinition.md [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+
+### Eligibility & Prerequisites
+
+This service is offered to BC Government development teams building cloud native applications on the OpenShift 4 Platform.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:32]:
```suggestion
This service is currently offered to BC Government development teams building cloud native applications on the OpenShift 4 Platform. 
```

File: docs/EDB/ServiceDefinition.md [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+
+You can find further information on the operator's features and functionality here: https://docs.enterprisedb.io/cloud-native-postgresql
+
+Associated with the operator is the RemoteDBA (RDBA) support service, also offered by EDB. 
+They are involved in providing support for the operator and support for database clusters belonging to teams which have purchased their additional support packages.
+
+### Eligibility & Prerequisites
+
+This service is offered to BC Government development teams building cloud native applications on the OpenShift 4 Platform. 
+The EnterpriseDB Service is available to all teams who have purchased a license.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:33]:
```suggestion
The EnterpriseDB Service is available to all the product teams **who have purchased a license from the EnterpriseDB vendor**. 
```

File: docs/EDB/ServiceDefinition.md [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+
+# BC Government EDB Operator Service Definition
+
+## Service Description
+
+### Summary
+
+### Features & Functions
+
+The Enterprise DB service is provided as one of the Shared Services on the BC Gov's DevOps OpenShift 4 Platform.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:21]:
```suggestion
The EnterpriseDB (EDB) service is provided as one of the Shared Services on the BC Gov's DevOps OpenShift 4 Platform. 
```

File: docs/EDB/ServiceDefinition.md [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+If a team wishes to purchase a license, they should contact [Olena Mitovska](mailto:olena.mitovska@gov.bc.ca) in order to discuss options - 
+licenses are purchased collectively for ease of administration.
+
+### Availability
+
+The operator is available to permit installation at all hours on the Silver cluster. Availability on Gold is to follow.
+Teams with RDBA support should contact them first, in the event of an issue with the operator. 
+Hours of availability for RDBA depend on your team's SLA with them.
+
+If something goes wrong with the operator that requires the platform team's intervention, platform support is available during business hours.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:53]:
```suggestion
If something goes wrong with the operator that requires the Platform Services Team's intervention, platform support is available during business hours.
```

File: docs/EDB/ServiceDefinition.md [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+
+Teams do not need to request anything from the platform team in order to make use of this operator. 
+Licenses are required in order to ensure that your database maintains its HA status after 30 days.
+If a team wishes to purchase a license, they should contact [Olena Mitovska](mailto:olena.mitovska@gov.bc.ca) in order to discuss options - 
+licenses are purchased collectively for ease of administration.
+
+### Availability
+
+The operator is available to permit installation at all hours on the Silver cluster. Availability on Gold is to follow.
+Teams with RDBA support should contact them first, in the event of an issue with the operator.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:50]:
```suggestion
Teams with RDBA support should contact the the EDB's RDBA team first, in the event of an issue with the operator. 
```

File: docs/EDB/ServiceDefinition.md [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+They will require some access to your project in order to provide full support for your database -
+each team is welcome to determine how much access they are willing to provide.
+Teams who have purchased this type of support should turn to RDBA first with any problem regarding their database.
+
+Should RDBA determine that the problem requires elevated privileges, teams are encouraged to work with RDBA to provide that support where possible.
+If, for example, the RDBA team requires admin privileges on a namespace but your team has determined that they will not provide that level of access,
+an admin from the your team should work directly with RDBA to take the necessary action.
+
+If the elevated privileges required to deal with the problem are beyond what the project team can provide, 
+please contact the platform services team directly.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:72]:
```suggestion
please contact the Platform Services team directly in #devops-ocp4-migration channel in Rocketchat. 
```

File: docs/EDB/ServiceDefinition.md [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+Teams who have purchased this type of support should turn to RDBA first with any problem regarding their database.
+
+Should RDBA determine that the problem requires elevated privileges, teams are encouraged to work with RDBA to provide that support where possible.
+If, for example, the RDBA team requires admin privileges on a namespace but your team has determined that they will not provide that level of access,
+an admin from the your team should work directly with RDBA to take the necessary action.
+
+If the elevated privileges required to deal with the problem are beyond what the project team can provide, 
+please contact the platform services team directly. 
+It is very unlikely that the platform services team should be involved in any issue that is not directly related to the operator itself.
+Please be sure that this level of cluster-wide privileges are needed before contacting platform services.
```
Comment (mitovskaol) [inline docs/EDB/ServiceDefinition.md:74]:
```suggestion
Please ensure that this level of cluster-wide privileges are needed, before contacting the Platform Services team.
```

**Review by mitovskaol — APPROVED** [review]

**Pull request merged**